packages/office-ui-fabric-react/src/components/TextField/examples/TextField.Controlled.Example.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { TextField } from 'office-ui-fabric-react/lib/TextField';
+import { Stack } from 'office-ui-fabric-react/lib/Stack';
+
+export interface ITextFieldControlledExampleState {
+  value1: string;
+  value2: string;
+}
+
+export class TextFieldControlledExample extends React.Component<{}, ITextFieldControlledExampleState> {
+  public state: ITextFieldControlledExampleState = { value1: '', value2: '' };
+
+  public render() {
+    // In a controlled field, rather than
+
+    return (
+      <Stack gap={15}>
+        <TextField
+          label="Basic controlled TextField"
+          value={this.state.value1}
+          onChange={this._onChange1}
+          styles={{ fieldGroup: { width: 300 } }}
+        />
+
+        <TextField
+          label="Controlled TextField limiting length of value to 5"
+          value={this.state.value2}
+          onChange={this._onChange2}
+          styles={{ fieldGroup: { width: 100 } }}
+        />
+      </Stack>
+    );
+  }
+
+  private _onChange1 = (ev: React.FormEvent<HTMLInputElement>, newValue?: string) => {
+    this.setState({ value1: newValue || '' });
+  };
+
+  private _onChange2 = (ev: React.FormEvent<HTMLInputElement>, newValue?: string) => {
+    if (!newValue || newValue.length <= 5) {
+      this.setState({ value2: newValue || '' });
+    } else {
+      // This block should NOT be necessary, but there's currently a bug (#1350) where a controlled
```

Comment:
This is the major downside to adding a controlled example right now...the TextField doesn't actually behave right when controlled, and fixing that would be such a major behavior change that I'm hesitant to do it outside a major version bump. So there's this extra block and giant comment needed to get the right behavior.